src/Client/index.ts

```diff
@@ -392,7 +391,7 @@ export class Client {
       return [true, error.leader];
     }
 
-    return [error instanceof UnavailableError || error instanceof TimeoutError];
```

Comment:
**QUESTION:** Is the `Abort` error thrown by the server is mapped somehow to `TimeoutError`?

Reply:
Yes: 

https://github.dev/EventStore/EventStore-Client-NodeJS/blob/49ba45ab614c10f46b4c4b929dd848a4c464912b/src/utils/CommandError.ts#L426-L428

Reply:
Thanks 👍